proofpoint_tap/komand_proofpoint_tap/actions/url_decode/action.py

```diff
@@ -12,8 +13,18 @@ def __init__(self):
         )
 
     def run(self, params={}):
+        params_urls = params.get(Input.URLS, [])
+        urls: [str] = []
+        for url in params_urls:
+            try:
+                encoded_url = quopri.decodestring(url).decode("UTF-8")
+            except UnicodeDecodeError as _:
```

Comment:
```suggestion
            except UnicodeDecodeError:
```
No need to use _ to ignore catching the exception variable